assets/src/components/parts/janus-dropdown/DropdownAuthor.tsx

```diff
@@ -57,7 +58,9 @@ const DropdownAuthor: React.FC<AuthorPartComponentProps<DropdownModel>> = (props
         id={`${id}-select`}
         value={-1}
         className={'dropdown '}
-        disabled={true}
+        onMouseDown={(e) => {
```

Comment:
Removing the '`disabled`' attribute so that the drop can be selected for drag n drop when user clicks on the dropdown control.

Adding '`onMouseDown`' so that when user clicks on component to drag it, dropdown should not get open or expanded.